src/Microsoft.ComponentDetection.Detectors/poetry/Contracts/PoetryPackage.cs

```diff
@@ -15,5 +16,9 @@ public class PoetryPackage
 
         [SuppressMessage("StyleCop.CSharp.NamingRules", "SA1300:ElementMustBeginWithUpperCaseLetter", Justification = "Deserialization contract. Casing cannot be overwritten.")]
         public PoetrySource source { get; set; }
+
+        public TomlTable dependencies { get; set; }
+
+        public TomlTable extras { get; set; }
```

Comment:
This needs a `[DataMember(Name = "extras")]` attribute.

```suggestion
        [DataMember(Name = "extras")]
        public TomlTable Extras { get; set; }
```